backend/back/src/migrations/1698521602687-WorkflowAreaReturnParams.ts

```diff
@@ -0,0 +1,57 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+type AreaParametersReturnFlow = {
+	areas: string[];
+	parameters: string[];
+};
+
+export class WorkflowAreaReturnParams1698521602687 implements MigrationInterface {
+	private readonly areaParametersReturnFlow: AreaParametersReturnFlow[] = [
+		{
+			areas: ["close-issue", "create-issue", "reopen-issue"],
+			parameters: ["url"],
+		},
+		{
+			areas: ["on-issue-close", "on-issue-create", "on-issue-reopen"],
+			parameters: ["author", "body", "createdAt", "title", "url"],
+		},
+		{
+			areas: ["on-pull-request-close", "on-pull-request-create", "on-pull-request-merge"],
+			parameters: ["author", "base", "body", "createdAt", "compare", "title", "url"],
+		},
+		{
+			areas: ["on-commit"],
+			parameters: ["author", "branch", "createdAt", "message", "url"],
+		},
+		{
+			areas: ["on-new-video"],
+			parameters: ["createdAt", "description", "title", "url"],
+		},
+		{
+			areas: ["seconds-interval"],
+			parameters: ["seconds"],
+		},
+	];
+
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		await queryRunner.query(`
+            alter table area
```

Comment:
lowercase SQL vibes

Reply:
ssshhhhhh, you are talking too loud, use sql not the uppercase version (⚠ too loud ⚠)